src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+    <div className="w-screen bg-subPupple shadow-convex">
```

Comment:
```suggestion
    <footer className="w-screen bg-subPupple shadow-convex">
```
div보다 적절한 footer tag를 활용해주시면 좋을 것 같습니다.
시멘틱 태그에 대해서 공부해보세요

[시멘틱태그_참고영상](https://www.youtube.com/watch?v=T7h8O7dpJIg&t=2s)